lib/shared/models/flash-state.js

```diff
@@ -123,9 +123,7 @@ exports.setProgressState = (state) => {
 
       speed: _.attempt(() => {
         if (_.isNumber(state.speed) && !_.isNaN(state.speed)) {
-          // Preserve only two decimal places
-          const PRECISION = 2
-          return _.round(units.bytesToMegabytes(state.speed), PRECISION)
+          return units.bytesToClosestUnit(state.speed)
```

Comment:
Rather than appending "/s" elsewhere, perhaps it'd make more sense to do that here?
I believe this would also mean that the store would be initialised to `speed: '0 B/s'` which is probably a more meaningful 'speed' value than `0 B`? ;-)

Reply:
Done this now.